tests/CMakeLists.txt

```diff
@@ -107,6 +107,10 @@ endif()
 # the risk of a race.
 add_custom_target(test_suite_bignum_generated_data DEPENDS ${bignum_generated_data_files})
 add_custom_target(test_suite_psa_generated_data DEPENDS ${psa_generated_data_files})
+# If SKIP_TEST_SUITES is not defined with -D, get it from the environment.
+if((NOT DEFINED SKIP_TEST_SUITES) AND (DEFINED ENV{SKIP_TEST_SUITES}))
```

Comment:
That's better, but still inconvenient. Any idea on [obeying it at runtime](https://github.com/Mbed-TLS/mbedtls/issues/6686)?

Reply:
I think it might be possible if we tell cmake to use `run_test_suites.pl` as make does currently, but I don't think there's a way to tell `ctest` to skip tests based on test-runtime values of environment variables.